libs/ledgerjs/packages/hw-transport/src/Transport.ts

```diff
@@ -301,9 +301,7 @@ export default class Transport {
 
     if (this.exchangeBusyPromise) {
       tracer.trace("Atomic exchange is already busy");
-      throw new TransportRaceCondition(
-        "An action was already pending on the Ledger device. Please deny or reconnect.",
```

Comment:
mmh actually this is interesting to keep the generic message for other consumer of this library: a lot of external wallets / metamask etc / , in order to show users understandable message.

it's ok to do our own i18n over it, but i think we should keep the generic message here.

( cc @mbertin-ledger for ownership )

Reply:
Changed